Framework/Core/include/Framework/ConfigParamRegistry.h

```diff
@@ -12,6 +12,7 @@
 
 #include "Framework/ParamRetriever.h"
 #include "Framework/ConfigParamStore.h"
+#include "Framework/Traits.h"
```

Comment:
What do you need from Traits?

Reply:
`is_base_of_template` to check for a `vector<T>`